indra/newview/installers/windows/installer_template.nsi

```diff
@@ -26,7 +26,6 @@
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;; Compiler flags
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
-Unicode true

```

Comment:
whats the story here?  is this needed?  is it temporary until we have a workaround for some problem?

Reply:
The `Unicode` parameter has been dropped from newer NSIS versions as `Unicode true` is default behavior. We have a couple options here:

1. Update TeamCity agents' NSIS installer
2. Perform some file manipulation on installer_template.nsi either in TC or Github to patch up the file for the respective NSIS installer version
3. Provide separate installer_template files...

My preference would be #1.